model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
```

Comment:
I wonder if we need `repository` in the name. Would `vcs.change_request.count` work?
I feel it'd be as descriptive, but shorter and easier to find.

Thoughts?

PS: I just realized there is an existing `vcs.repositry.change.*` namespace, but then the question applies to it as well - do we really need full hierarchy in the name or can shorten it to `vcs.change_request.*`

Reply:
We discussed this in today's CICD SIG meeting.

The conclusion was to rename `vcs.repositry.change.*` to `vcs.change.*`.
It is indeed as descriptive, shorter and easier to find.

We decided to stick with `change` and not `change_request`.
According to @adrielp we had this discussion in https://github.com/open-telemetry/semantic-conventions/pull/1075 about `change` vs `change_request` but we couldn't find an exact link during our SIG.
The term `change` aligns more closely with CDFoundation definition: https://bestpractices.cd.foundation/learn/ci/#heading-def
Also not all VCS refer to changes as request, there are also changelist and patches.

I implemented this rename in 536a2a7defc49a81e598b228848258ff5201e94f.